Doc/library/asyncio-task.rst

```diff
@@ -1222,7 +1222,7 @@ Task Object
    To cancel a running Task use the :meth:`cancel` method.  Calling it
    will cause the Task to throw a :exc:`CancelledError` exception into
    the wrapped coroutine.  If a coroutine is awaiting on a Future
-   object during cancellation, the Future object will be cancelled.
+   or Task object during cancellation, the awaited object will be cancelled.
```

Comment:
It actually supports any future-like objects, you can say it like "if coroutine is awaiting on future-like object then that will be cancelled", with sphinx markers as necessary